scss/utilities/_u--line-limit.scss

```diff
@@ -0,0 +1,15 @@
+// ========================================================
+// @utility: line-limit
+// ========================================================
+//
+
+// component config
+//
+$line-limit-width: $spacing--md * 37;
```

Comment:
@adamgparsons where did this number come from, could we comment this line so that we know for future ref?
Ta.

Reply:
Nice, that makes perfect sense, 👍. 
This resolves #51